### ISSUE TITLE
id праздника не везде проставлены

### ISSUE THREAD

**Issue by MainDalen** [xmlcalendar/data#36]
Добрый день!
Спасибо за Вашу работу,  уже не первый раз пользуюсь!
Подскажите, пожалуйста, можно ли везде проставить id праздника?
Или хотя бы только на выходных днях с t=1
Столкнулся сейчас с тем, что не получается со 100% точностью раскидать выходные по праздникам из-за этих пустот
Готов самостоятельно проставить id и отправить pull request, от Вас прошу только зеленый свет :)

**Comment by xmlcalendar** [comment]
Добрый день,Для t=1 не проставляются праздники если это перенос. Вы не могли бы выслать пример xml где вы хотите проставить?17:55, December 7, 2022, Dalen Mainerman ***@***.***>:
Добрый день!
Спасибо за Вашу работу,  уже не первый раз пользуюсь!
Подскажите, пожалуйста, можно ли везде проставить id праздника?
Или хотя бы только на выходных днях с t=1
Столкнулся сейчас с тем, что не получается со 100% точностью раскидать выходные по праздникам из-за этих пустот
Готов самостоятельно проставить id и отправить pull request, от Вас прошу только зеленый свет :)

**Comment by MainDalen** [comment]
Хотел проставить для праздников РФ
Например, 2018:
https://github.com/xmlcalendar/data/blob/master/ru/2018/calendar.xml
30.04.2018-02.05.2018 относятся к первому мая, но поле h пустое, информации о переносе тут тоже нет

**Comment by xmlcalendar** [comment]
Да, раньше переносы не проставлялись, поэтому вы видите выходной без h атрибута.Правильней было бы для них проставить перенос, т.е. 2 мая не является праздником, это перенос с 7 января.18:26, December 7, 2022, Dalen Mainerman ***@***.***>:
Хотел проставить для праздников РФ
Например, 2018:
https://github.com/xmlcalendar/data/blob/master/ru/2018/calendar.xml
30.04.2018-02.05.2018 относятся к первому мая, но поле h пустое, информации о переносе тут тоже нет
